modules/minigl/src/main/java/org/jpos/gl/tools/Import.java

```diff
@@ -53,11 +53,27 @@
 public class Import implements EntityResolver {
     Log log = LogFactory.getLog (Import.class);
     private static final String URL = "http://jpos.org/";
+    /**
+     * This setting controls whether to check that child account codes
+     * contain the parent account code as a prefix. Defaults to true
+     * for optimal compatibility with getBalances native queries,
+     * but may be set to false to import a legacy system that doesn't
+     * use native queries.
+     */
+    private Boolean strictAccountCodes = true;
```

Comment:
Not a big deal, but I believe a native `boolean` would be OK here.

Reply:
Thanks, fixed